packages/mineral-ui-tokens/README.md

```diff
@@ -13,79 +12,93 @@ or
 yarn add mineral-ui-tokens
 ```
 
-
 ## Usage
-_This package uses the same [import syntax as the mineral-ui library][import-syntax]._
 
-Token names use a `[target]_[property]_[variation]_[state]` naming scheme.
+> All Tokens and their corresponding values can be viewed on the
+[Mineral UI Tokens][token-page] page.
 
-- `target` - the type of element targeted by the token, e.g. "input" or "panel"
-- `property` (required) - the CSS property of the token, e.g. "backgroundColor"
-- `variation` - any differentiating aspect of the token that isn't state, e.g. "brand", "primary", "success"
-- `state` - state-dependent aspects, e.g. "focus", "selected"
+_This package uses the same
+[import syntax as the mineral-ui library][import-syntax]._
 
+Token names use a `[target]_[property]_[variation]_[state]` naming scheme.
+
+* `target` - the type of element targeted by the token, e.g. "input" or "panel"
+* `property` (required) - the CSS property of the token, e.g. "backgroundColor"
+* `variation` - any differentiating aspect of the token that isn't state, e.g.
+"brand", "primary", "success"
+* `state` - state-dependent aspects, e.g. "focus", "selected"
 
 ### JavaScript
 
 JavaScript token names are formatted in "pseudo_camelCase", e.g. `boxShadow_1`
 
-
 #### Import tokens from the default export:
+
 ```js
 import tokens from 'mineral-ui-tokens';
 ```
 
-#### Import the palette, specific [color ramps][colorPage], or specific tokens from named exports:
+#### Import the palette, specific [color ramps][color-page], or specific tokens from named exports:
+
 ```js
 import { palette, magenta, boxShadow_1 } from 'mineral-ui-tokens';
 ```
 
-#### Import tokens & palette, as Scss variables:
+#### Import tokens & palette, as Sass variables:
+
 ```js
 import 'mineral-ui-tokens/index.scss';
```

Comment:
The `index.scss` file is not being included in the npm package.  It can be included by updating the last line of the build script as follows.

```js
exec('cp {README.md,LICENSE.md,src/index.scss} dist');
```

Reply:
Great catch, thanks!